sphinx_rtd_theme/locale/sphinx.pot

```diff
@@ -0,0 +1,162 @@
+# Translations template for sphinx_rtd_theme.
+# Copyright (C) 2018 ORGANIZATION
+# This file is distributed under the same license as the sphinx_rtd_theme
+# project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2018.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: sphinx_rtd_theme 0.3.0\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2018-04-16 18:43-0400\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.5.3\n"
+
+#: sphinx_rtd_theme/breadcrumbs.html:31
+msgid "Docs"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:43 sphinx_rtd_theme/breadcrumbs.html:45
+msgid "Edit on GitHub"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:50 sphinx_rtd_theme/breadcrumbs.html:52
+msgid "Edit on Bitbucket"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:57 sphinx_rtd_theme/breadcrumbs.html:59
+msgid "Edit on GitLab"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:62 sphinx_rtd_theme/breadcrumbs.html:64
+msgid "View page source"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:74 sphinx_rtd_theme/footer.html:5
+msgid "Next"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:77 sphinx_rtd_theme/footer.html:8
+msgid "Previous"
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:19
+#, python-format
+msgid "&copy; <a href=\"%(path)s\">Copyright</a> %(copyright)s."
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:21
+#, python-format
+msgid "&copy; Copyright %(copyright)s."
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:26
+#, python-format
+msgid ""
+"\n"
+"        <span class=\"build\">\n"
+"          Build\n"
+"          <a href=\"%(build_url)s\">%(build_id)s</a>.\n"
+"        </span>\n"
+"      "
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:33
+#, python-format
+msgid ""
+"\n"
+"        <span class=\"commit\">\n"
+"          Revision <code>%(commit)s</code>.\n"
+"        </span>\n"
+"      "
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:39
+#, python-format
+msgid "Last updated on %(last_updated)s."
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:46
+msgid ""
```

Comment:
Urls in translation message? can we get around this?

Reply:
@Blendify, you are right! fixed this string :+1: thanks for your feedback :nerd_face: